it/uk/gov/hmrc/agentrelationships/controllers/RelationshipsControllerTestOnlyISpec.scala

```diff
@@ -27,11 +27,10 @@ import uk.gov.hmrc.play.test.UnitSpec
 
 import scala.concurrent.ExecutionContext.Implicits.global
 
-class RelationshipTestOnlyISpec extends UnitSpec
+class RelationshipsControllerTestOnlyISpec extends UnitSpec
   with MongoApp
   with OneServerPerSuite
   with WireMockSupport
-  with GovernmentGatewayProxyStubs
```

Comment:
Still references "microservice.services.government-gateway-proxy.port" down in Guice configuration